### PR TITLE
Added simple WebXR playground URLs

### PR DESCRIPTION
**Pull request by flushpot1125**
Hi contributors!
There are a few simple playground in WebXR section. My pull requests propose to add two PG. (1) Attach child mesh on controllers, (2) Simply grabbing objects on controllers. I hope these simple PG would be helpful for WebXR-BJS new comers.
Any comments and feedbacks are welcome!

### CODE REVIEW

**Review by RaananW — COMMENTED** [review]
I don't have a VR device next to me at the moment, so I can't check the demos for the time being. 
I think it is not needed to stat that a VR device is needed, you just need to add "VR" to the playground description. Also, if possible, can you move the demos to the end of the list instead of the beginning of the list?

I will test the demos as soon as I can and will approe after.

Thank you so much! :-)

**Comment by flushpot1125** [comment]
Hi @RaananW 
Thanks for reviewing. I modified the order of PG and deleted "VR".
Could you check again? Thanks for your reviewing again.

File: content/How_To/WebXR/WebXR_Demos_and_Examples.md [hunk 1]
```diff
@@ -69,15 +69,13 @@ panel.addControl(picker);
 <Playground id="#9K3MRA#2" title="WebXR Color Picker" description="Simple WebXR color picker example." isMain={true} category="VR/AR"/>
 
 ## Other demos
-
-VR devices are required.  
+ 
+<Playground id="#PPM311#63" title="Goalkeeper Training" description="Goalkeeper Training" isMain={true} category="VR/AR"/>
+<Playground id="#B922X8#19" title="Physics Playground" description="Physics Playground" isMain={true} category="VR/AR"/>
 
 <Playground id="#F41V6N#139" title="A cylinder object is child of a controller" description="A cylinder object is child of a controller" isMain={true} category="VR/AR"/>
```
Comment (RaananW) [inline content/How_To/WebXR/WebXR_Demos_and_Examples.md]:
```suggestion
<Playground id="#F41V6N#139" title="A cylinder object is child of a controller" description="A cylinder object is child of a controller"/>  
```

File: content/How_To/WebXR/WebXR_Demos_and_Examples.md [hunk 2]
```diff
@@ -69,10 +69,14 @@ panel.addControl(picker);
...
 
 ## Other demos
-
+ 
 <Playground id="#PPM311#63" title="Goalkeeper Training" description="Goalkeeper Training" isMain={true} category="VR/AR"/>
 <Playground id="#B922X8#19" title="Physics Playground" description="Physics Playground" isMain={true} category="VR/AR"/>
 
+<Playground id="#F41V6N#139" title="A cylinder object is child of a controller" description="A cylinder object is child of a controller"/>  
+
+<Playground id="#1FTUSC#29" title="Simply grabbing objects by controllers" description="Simply grabbing objects by controllers" isMain={true} category="VR/AR"/>
```
Comment (RaananW) [inline content/How_To/WebXR/WebXR_Demos_and_Examples.md]:
```suggestion
<Playground id="#1FTUSC#29" title="Simply grabbing objects by controllers" description="Simply grabbing objects by controllers"/> 

```

**Review by RaananW — APPROVED** [review]

**Pull request merged**